pkgs/development/libraries/zint/default.nix

```diff
@@ -4,35 +4,41 @@
 , wrapQtAppsHook
 , cmake
 , qtbase
+, qtsvg
 , qttools
-, nix-update-script
+, testers
+, zint
 }:
 
 stdenv.mkDerivation rec {
   pname = "zint";
-  version = "2.11.0";
+  version = "2.11.1";
 
   src = fetchFromGitHub {
-    owner = "woo-j";
+    owner = "zint";
     repo = "zint";
     rev = version;
-    sha256 = "sha256-DtfyXBBEDcltGUAutHl/ksRTTYmS7Ll9kjfgD7NmBbA=";
+    sha256 = "sha256-ozhXy7ftmGz1XvmF8AS1ifWJ3Q5hLSsysB8qLUP60n8=";
   };
 
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [ cmake wrapQtAppsHook ];
 
-  buildInputs = [ qtbase qttools ];
+  buildInputs = [ qtbase qtsvg qttools ];
+
+  cmakeFlags = [ "-DZINT_QT6:BOOL=ON" ];
 
   postInstall = ''
-    install -Dm644 $src/zint-qt.desktop $out/share/applications/zint-qt.desktop
-    install -Dm644 $src/zint-qt.png $out/share/pixmaps/zint-qt.png
-    install -Dm644 $src/frontend_qt/images/scalable/zint-qt.svg $out/share/icons/hicolor/scalable/apps/zint-qt.svg
+    install -Dm644 -t $out/share/applications $src/zint-qt.desktop
+    install -Dm644 -t $out/share/pixmaps $src/zint-qt.png
+    install -Dm644 -t $out/share/icons/hicolor/scalable/apps $src/frontend_qt/images/scalable/zint-qt.svg
```

Comment:
```suggestion
    install -Dm644 -t $out/share/applications zint-qt.desktop
    install -Dm644 -t $out/share/pixmaps zint-qt.png
    install -Dm644 -t $out/share/icons/hicolor/scalable/apps frontend_qt/images/scalable/zint-qt.svg
```

Reply:
It doesn't work. These files are not there during `installPhase`.